### ISSUE TITLE
host Epiviz on new Google cloud instance

### ISSUE THREAD

**Issue by RLC-DCPPC** [nemoarchive/analytics#35]

**Comment by jkanche** [comment]
@jorvis or Anup,

should epiviz for nemo analytics run on a new google cloud instance because of the billing you were talking about on the last call ?

**Comment by jorvis** [comment]
Yes, it should.  I just added you to the nemo-analytics gcloud project so you can create an instance there.

**Comment by jkanche** [comment]
@jorvis, I might need some help from you on this,

- [ ] I tried to downgrade the storage on the epiviz instance. Since everything will be file based, we don't need 400Gb of disk. But it won't let me change it. 
- [ ] How do I setup a subdomain to the instance. Something similar to what we did in gEAR,  epiviz.nemoarchive.org ?

**Comment by jorvis** [comment]
For the first one, you can only resize disks if you first shut down the instance.  Did you try that? 

To set up the subdomain we need to make sure the instance is using a statically-assigned IP address and then have Victor do a DNS entry for the subdomain.  I could do it on the gear pages because I manage the domain, but I don't for nemoarchive.

**Comment by jkanche** [comment]
yea, I tried shutting it down, but it won't let me set a value below 400

**Comment by jorvis** [comment]
Relevant [article](https://blog.kevinlin.info/post/google-compute-engine-reducing-the-size-of-a-persistent-disk).
